src/MIDebugPackage/MIDebugPackagePackage.cs

```diff
@@ -37,6 +37,7 @@ namespace Microsoft.MIDebugPackage
     // This attribute is needed to let the shell know that this package exposes some menus.
     [ProvideMenuResource("Menus.ctmenu", 1)]
     [Guid(GuidList.guidMIDebugPackagePkgString)]
+    [ProvideBindingPath]
```

Comment:
To make sure I understand, this is needed because some Linux workload dll is referencing some MIEngine dll?

I think this needs to be under `#if LAB`, and we should probably have a comment explaining why we have it.

Reply:
Modified to:
#if LAB
    // Adding this dll location to VS probe path. Custom launcher types are referenced by Linux/Azure Sphere workloads. 
    [ProvideBindingPath]
#endif